pisa/stages/flux/mceq_barr.py

```diff
@@ -241,6 +247,18 @@ def setup_function(self):
         # Encoding is to support pickle files created with python v2
         self.spline_tables_dict = pickle.load(BZ2File(spline_file), encoding="latin1")
 
+        # Ensure that the user is not loading an incompatible spline
```

Comment:
I think to remember that this linking had to be disabled in the past for this stage due to some observed differences, which were unresolved at the time. Maybe this has now gone away with pisa4 though? Would be good to test

Reply:
The linking in line 184-193? What were the observed differences? I can take a look. I added these lines comparing to [the reduced barr stage](https://github.com/icecube/pisa/blob/master/pisa/stages/flux/mceq_barr_red.py), which also has the linking.

Reply:
Hmm, maybe my memory is tricking me. Will merge this then